doc/release-notes/7440-multilicense.md

```diff
@@ -0,0 +1,119 @@
+
+
+## Major Use Cases and Infrastructure Enhancements
+
+
+- When creating/updating datasets, users can select from a set of standard licenses configured by the administrator or provide custom terms (if the installation is configured to allow them).
+
+## Notes for Dataverse Installation Administrators
+
+### Updating for multiple license support
+
+As part of installing/upgrading an existing installation, administrators may wish to add additional license choices and/or configure Dataverse to allow custom terms. Adding additional licenses is managed via API. Licenses are described via a JSON structure providing a name, URL, short description, and optional icon URL. Additionally licenses may be marked as active (selectable for new/updated datasets) or inactive (only allowed on existing datasets) and one license can be marked as the default. Custom Terms are allowed by default (backward compatible with the current option to select 'No' to using CC0) and can be disabled by setting `:AllowCustomTermsOfUse` to false.
+
+Further administrators should review the following automated migration of existing licenses and terms into the new license framework and, if desired, should manually find and update any datasets for which the automated update is problematic. 
+To understand the migration process, it is useful to understand how the multiple license feature works in this release:
+
+'Custom Terms', aka a custom license, are defined through entries in the following fields of the dataset "Terms" tab:
+- Terms of Use
+- Confidentiality Declaration 
+- Special Permissions 
+- Restrictions 
+- Citation Requirements 
+- Depositor Requirements 
+- Conditions 
+- Disclaimer
+
+'Custom Terms' require,at a minimum, a non-blank entry in the "Terms of Use" field. Entries in other fields are optional.
+
+Since these fields are intended for terms/conditions that would potentially conflict with or modify the terms in a standard license, they are no longer shown when a standard license is selected.
+
+In earlier Dataverse releases, it was possible to select the CC0 license and have entries in the fields above. It was also possible to say 'No' to using CC0 and leave all of these terms fields blank. 
+
+The automated process will update existing datasets as follows.
+
+- 'CC0 Waiver' and no entries in the fields above -> CC0 License (no change)
```

Comment:
This is a tricky one. I think Waiver made sense when the only alternative was to add custom terms/not accept the waiver, but now with the overall option being 'Licenses' having one of them labelled as a Waiver may be more confusing than it is worth. 

There are more comments about naming below, but I'll note here that the intent was not to replace screen text 'CC0 Waiver' with 'CC0 License' - it would just be 'CC0' as the value where the License/DUA choice is shown  (DUA == data user agreement). Nominally the important connection for a license is the URL since that gets you to the text of the license, so the name could potentially be non-standard, i.e. it could be 'CC0 Waiver' if we wanted that as a default or if individual sites wanted more backward compatibility. (Not sure if one would then add 'CC-By License' instead of 'CC-By' etc.)

I think where the code is now is trying to use the standard community name (perhaps confused with identifier still in some places - more below), which would presumably be used if/when licenses get tied to an external service and/or if internationalization is added.